terraform/modules/vpc-nacls/main.tf

```diff
@@ -35,7 +35,7 @@ resource "aws_network_acl" "protected" {
   )
 }
 
-#tfsec:ignore:aws-vpc-no-excessive-port-access tfsec:ignore:aws-ec2-no-public-ingress-acl
+#trivy:ignore:AVD-AWS-0102
```

Comment:
it would be useful if we added the information why the warning is being suppressed

Reply:
Comment added to trivy ignores